src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -423,7 +435,9 @@ JSON example:
 ```JSON
 {
   "PhaseEncodingDirection": "j-",
-  "TotalReadoutTime": 0.095
+  "TotalReadoutTime": 0.095,
+  "BigDelta": 0.005,
+  "SmallDelta": 0.0005
```

Comment:
I could not find the definitions of those fields. Was it introduces in other PR?

Reply:
@mattcieslak added a table as for other definitions - marking resolved.